components/integrations/components/abstract_incoming_webhook.jsx

```diff
@@ -219,6 +235,64 @@ export default class AbstractIncomingWebhook extends React.Component {
                                 </div>
                             </div>
                         </div>
+                        { global.mm_config.EnablePostUsernameOverride === 'true' &&
```

Comment:
I'd have liked to access the global config via Redux, but opted not to tackle that as part of this PR given that there appears to be no existing reducers for same.

Reply:
Well we are converting everything to be pureComponents / funtional components and redux connected. You have access in state.entities.general.config so feel free to convert it, actually I would prefer if you do

Reply:
It should be under `state.entities.general.config`

Reply:
Oh -- thanks for sharing this! I should have browser the redux state tree itself. I'll update this accordingly.